.github/actions/migration_tests/action.yml

```diff
@@ -45,6 +45,21 @@ runs:
         COMPOSE_PROJECT_NAME: "docker-compose"
       env:
         COMPOSE_PROJECT_NAME: "docker-compose"
+    - name: "Bring composer down again"
+      shell: bash
+      run: breeze down
+      env:
+        COMPOSE_PROJECT_NAME: "docker-compose"
+    - name: "Test offline migration ${{env.BACKEND}}"
+      shell: bash
+      run: >
+        breeze shell "airflow db reset -y &&
+        airflow db downgrade -n 2.7.0 -y &&
+        airflow db migrate -s"
+        COMPOSE_PROJECT_NAME: "docker-compose"
```

Comment:
```suggestion
```

Reply:
This was copied from a similar job above. Not sure what it does. I will remove it and see